src/main/resources/dse_protocol_v1.spec

```diff
@@ -330,7 +330,8 @@ Table of Contents
 
   Asks the server to return which STARTUP options are supported. The body of an
   OPTIONS message should be empty and the server will respond with a SUPPORTED
-  message.
+  message. Since OPTIONS can come before a STARTUP request, as a consequence,
+  an OPTIONS message must never be compressed.
```

Comment:
We don't maintain these specs in this project, they are copied directly from the Cassandra/DSE codebase. So I think it would be better to suggest the change there, in case there are updates and this file gets overwritten.

That being said, section 5 kind of already says it:

> once the STARTUP frame has been received by the server, messages can be compressed